lib/core.ps1

```diff
@@ -18,6 +18,14 @@ $globaldir = $env:SCOOP_GLOBAL, "$env:ProgramData\scoop" | Select-Object -first
 #       Use at your own risk.
 $cachedir = $env:SCOOP_CACHE, "$scoopdir\cache" | Select-Object -first 1
 
+$configFile = "$env:USERPROFILE\.config\scoop\config.json"
```

Comment:
```suggestion
$configHome = $env:XDG_CONFIG_HOME, "$env:USERPROFILE\.config" | Select-Object -First 1
$configFile = "$configHome\scoop\config.json"
```

`$HOME/.config` is [`$XDG_CONFIG_HOME`](https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html), so check it first.